xpk.py

```diff
@@ -2609,21 +2585,13 @@ def run_gke_cluster_create_command(args, gke_control_plane_version: str) -> int:
   Returns:
     0 if successful and 1 otherwise.
   """
-  # cluster_cpu_machine_type is soon to be deprecated!
   machine_type = args.default_pool_cpu_machine_type
-  if args.cluster_cpu_machine_type != '':
```

Comment:
Lets depreciate this in a separate PR. Couldn't potentially cause issues if folks are still using this arg.

Reply:
For sure, we now have a lot of items to deprecate. Perhaps a PR targeting cleanup would be helpful!

Reply:
It looks like the code is still being removed in this PR?

Reply:
The change to revert this was lost in a stash :) 
Sorry about that!